lib/external/es6-promise-2.0.1.js

```diff
@@ -0,0 +1,960 @@
+/*!
```

Comment:
Can this be bower installed? https://github.com/jakearchibald/es6-promise/blob/master/bower.json